### ISSUE TITLE
Add attributes to WMT diagnostics

### ISSUE THREAD

**Issue by hdrake** [NOAA-GFDL/xwmt#46]
The WMT rate variables output by `xwmt` currently either have no attributes or–even worse–inherit attributes from the input budget diagnostics, meaning that the units and other metadata are wrong and misleading.

These should have descriptive attributes, at the very least explaining what each term is and what its units are (e.g. `kg s-1`).

**Comment by gmacgilchrist** [comment]
Agreed, improved documentation in general, and attributes attached to variables, would be nice. This should be easier since @hdrake's [paper](https://onlinelibrary.wiley.com/doi/abs/10.1029/2024MS004383) was published. We are back working on this more regularly in St Andrews, so can contribute some of these improvements.

FYI, @cspencerjones, for the negative LHS terms, see the derivations in Appendix A of the paper, arriving at equations A15-17. It's all about reverting from conservation equations for extensive quantities into those for intensive quantities (which is what's relevant for WMT).

**Comment by hdrake** [comment]
Just a quick note to let you know that I’m recovering from a broken collarbone so not writing much code right now. I’ll be checking GitHub to answer questions and comment on PRs though.

Like Graeme said, the “negative rhs” term is what people usually call the freshwater forcing term. I found it really confusing that people don’t usually explain where that term come from, so I went in a lot of detail for it. The reason it’s important is because it’s not actually a term in MOM6 tracer budgets so you need to add it to both side of the equation to maintain conservation.
